src/main/resources/application.yml

```diff
@@ -22,6 +22,12 @@ spring:
     maxActive: 20
     testOnBorrow: true
     validationQuery: SELECT 1
+
+hystrix.command.default.execution.isolation.strategy: SEMAPHORE
+hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds: 15000
+hystrix.command.default.circuitBreaker.requestVolumeThreshold: 20
+hystrix.command.default.circuitBreaker.sleepWindowInMilliseconds: 5000
```

Comment:
What these params mean exactly? That if 20 errors happen withing 5 sec - then it will break the circuit for 15 sec?